lib/datadog.py

```diff
@@ -71,6 +71,7 @@ def update_config(m2ee, app_name):
         return
 
     tags = buildpackutil.get_tags()
+    statsd_port = 8125 if not buildpackutil.is_appmetrics_enabled() else 8126
```

Comment:
I would rather we use an explicit if else here, instead of a conditional statement. (Also, the `if not` is less readable than simply using if`)

```suggestion
    if buildpackutil_is_appmetrics_enabled:
        statsd_port = 8126
    else:
        statsd_port = 8125
```